features/README.md

```diff
@@ -70,33 +34,4 @@ Structuur wordt aangegeven via de sleutelwoorden "Functionaliteit", "Achtergrond
 | Abstract Scenario | Scenario Outline  | Scenario sjabloon die gevallen uit voorbeelden doorloopt |
 | Voorbeelden       | Examples          | Tabel met testwaarden die met Abstract scenario worden doorlopen |
 
-De Gherkin testscenario's worden opgenomen in de map /features. Voor elke functionaliteit wordt een bestand gemaakt met als extensie ".feature". Per functionaliteit (feature-bestand) kunnen er meerdere scenario's zijn gedefinieerd om de betreffende functionaliteit.
-
-## Tests
```

Comment:
Dit stuk documentatie was voor mij erg nuttig, is dit ook verplaatst naar een andere plek?

Reply:
Beste @pimotte ik heb een deel van deze toelichting teruggezet. 

We hebben de tests (Behave) incl. de testdata verwijderd, omdat de features op dit moment niet echt te automatiseren zijn. Hiervoor ontbreekt vaak de testdata of is de feature beschrijving niet consistent met de eerder gemaakte testdata, zijn de omschrijvingen niet consequent genoeg opgezet, enz.